<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Filter</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>671</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
     <property name="leftMargin">
      <number>15</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="filterComboBox">
       <item>
        <property name="text">
         <string>Gaussian Filter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Box Filter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Median Filter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bilateral Filter</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="imageInput"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Image input</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="outputButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="inputButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="saveInput"/>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="showImageCheckBox">
       <property name="text">
        <string>Display image before save</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="runFilterButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
